LoginPage.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import LoginPage from './LoginPage';
+
+test('renders login form', () => {
+  const { getByLabelText } = render(<LoginPage />);
+  const emailField = getByLabelText(/email/i);
+  const passwordField = getByLabelText(/password/i);
+  expect(emailField).toBeInTheDocument();
+  expect(passwordField).toBeInTheDocument();
+});
+
+test('can submit form', () => {
+  const { getByLabelText, getByText } = render(<LoginPage />);
+  const emailField = getByLabelText(/email/i);
+  const passwordField = getByLabelText(/password/i);
+  const submitButton = getByText(/login/i);
+  fireEvent.change(emailField, { target: { value: 'test@test.com' } });
+  fireEvent.change(passwordField, { target: { value: 'password' } });
+  fireEvent.click(submitButton);
+  expect(submitButton).toBeDisabled();
```

Comment:
The test 'can submit form' does not assert anything. Consider adding assertions to check the form submission functionality.